login.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:firebase_database/firebase_database.dart';
+import 'package:flutter/material.dart';
+import 'package:math_go/screens/map_view.dart';
```

Comment:
For consistency, we should be importing our own stuff in the form `import 'screens/map_view.dart';` instead of with the package name (see other files for examples).